src/components/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC, useContext, useState } from 'react';
+import { TodoContext } from '../TodoContext';
+import { Todo } from '../types/Todo';
+
+export const Header: FC = () => {
+  const { todos, setTodos } = useContext(TodoContext);
+  const [query, setQuery] = useState('');
+
+  const addTodo = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const newTodo: Todo = {
```

Comment:
you shouldn't add a new todo if the input is empty or there are only spaces typed:
![image](https://github.com/mate-academy/react_todo-app/assets/86923089/ff9309e0-596b-4895-b7ca-c4f29d7761c2)